src/components/TheHeader.vue

```diff
@@ -14,14 +14,16 @@
           >
             {{ signupButtonText }}
           </v-btn>
-          <img
-            id="logo"
-            alt="Cuttle logo"
-            :src="`/img/cuttle_logo_text_${logoColor}.svg`"
-            width="60"
-            height="60"
-            class="ma-md-auto desktop-logo"
-          >
+          <router-link :to="{ name: ROUTE_NAME_HOME }" style="display: contents;">
```

Comment:
Generally `display: contents` is unideal for links, because we want screen readers to pick up that there is a link tag. Did you find that the layout was messed up without including it? If so we should look to jigger the styles so that the parent can be rendered normally & picked up by screen readers, while allowing it to look the same visually.

If it looks fine without setting the style on the `router-link`, then we can remove the style attribute here without making other changes
```suggestion
          <router-link :to="{ name: ROUTE_NAME_HOME }">
```

Reply:
I originally used `display: contents` because the router-link wrapper was pushing the logo down. I was unware of the accessibility issues, so thank you! I went ahead switched to using a logo-container class using position: absolute and transform to maintain the centered positioning. I hope this solution will do, and thank you for the welcome and the opportunity to learn🙏